BlueprintUICommonControls/Sources/AttributedLabel.swift

```diff
@@ -256,9 +273,8 @@ extension AttributedLabel {
                     layer.shouldRasterize = false
                 }
 
+                applyLinkColors()
```

Comment:
We only needed to do this when measuring as well.

Reply:
👁️ 👁️  good catch, I think I can move these too! (Though, not layout direction perhaps)